cli/internal/cmd/cmd.go

```diff
@@ -10,3 +10,53 @@ Package cmd provides the Constellation CLI.
 It is responsible for the interaction with the user.
 */
 package cmd
+
+import (
```

Comment:
nit: does it make sense to put this in `workspace.go`. Seems more intuitive to look for anything there.
Even more intuitive a type alias `workspace` with these methods? But I see that it might be too much work. Just a thought.

Reply:
Moving the function makes sense.
Not sure about introducing a new type alias though: Since the value of the workspace is not consumed by functions written by us, but by the standard library, afero, and others, there would be a lot of type conversions.
Additionally I don't see a big benefit other than indicating what the variables is used for in the type name.